node-sdk-sample/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "sdk-sample",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "commonjs",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint src",
+    "build": "tsc"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "ts-node": "^10.9.2"
+  },
+  "devDependencies": {
+    "@types/jest": "^29.5.14",
+    "@typescript-eslint/eslint-plugin": "^8.19.1",
+    "@typescript-eslint/parser": "^8.19.1",
+    "@zowe/core-for-zowe-sdk": "^8.10.4",
```

Comment:
Is the dev/peer dep on Core SDK needed? I don't see it imported anywhere.

I know that several of our SDKs depend on it (zosfiles, zosjobs, etc), but I think this is only for the `ZosmfRestClient`.